tests/go.mod

```diff
@@ -1,101 +1,85 @@
 module github.com/kubearmor/KubeArmor/tests
 
-go 1.18
-
-replace (
-	github.com/cilium/cilium => github.com/cilium/cilium v1.10.0
-	github.com/cilium/cilium-cli => github.com/cilium/cilium-cli v0.11.4
-	github.com/optiopay/kafka => github.com/cilium/kafka v0.0.0-20180809090225-01ce283b732b
-	k8s.io/api => k8s.io/api v0.22.3
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.22.3
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.22.3
-	k8s.io/client-go => k8s.io/client-go v0.22.3
-	sigs.k8s.io/controller-tools => github.com/christarazi/controller-tools v0.6.2
-)
+go 1.20
 
 require (
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorHostPolicy v0.0.0-20220823050108-4455a183e9ef
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorPolicy v0.0.0-20220823050108-4455a183e9ef
-	github.com/kubearmor/KubeArmor/protobuf v0.0.0-20220815044951-425f333210e1
-	github.com/kubearmor/kubearmor-client v0.8.3
-	github.com/onsi/ginkgo/v2 v2.6.0
-	github.com/onsi/gomega v1.24.0
-	github.com/sirupsen/logrus v1.9.0
-	google.golang.org/protobuf v1.28.1
-	k8s.io/api v0.24.3
-	k8s.io/apimachinery v0.24.3
-	k8s.io/cli-runtime v0.24.3
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/kubectl v0.24.2
+	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20230519075515-e8b45043a53d
+	github.com/kubearmor/KubeArmor/protobuf v0.0.0-20230519075515-e8b45043a53d
```

Comment:
Thoughts on whether we should be using a replace directive with local path for dependencies that are present in KubeArmor repo? :thinking: @rksharma95 @daemon1024